test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,6 +1,30 @@
+---
+# NXAPI is enabled differently on NX-OS platforms.
```

Comment:
Agreed, just wanted to make sure that action item didn't get lost. 

When fix the issue could you please revert this change thanks.